Rover/mode.cpp

```diff
@@ -414,6 +414,14 @@ float Mode::calc_speed_nudge(float target_speed, bool reversed)
 // this function updates _distance_to_destination
 void Mode::navigate_to_waypoint()
 {
+    if ( g2.sailboat.in_irons()){
+       // set rudder output to turn the boat in required direction
+       // rudder_percent ( 0.0f to 1.0f) sets the rudder angle used to turn the boat
```

Comment:
rudder_percent ?

Reply:
Good point. I will update the comment, which is out of date   The detail is out of place there anyway